README.md

```diff
@@ -11,3 +11,16 @@ Note that we currently use `ln -s` to "install" git hooks rather than just makin
 ```
 cd dev-tools && ./bootstrap
 ```
+
+## symlinks.ps1
+
+On `Windows` unix symlink doesn't not work as expected (check [this](https://github.com/3rdparty/eventuals/issues/153) issue). That's why we have been created `symlinks.ps1` PowerShell script. To run this script do the following:
+
+- Make sure if you are able to create symlinks on Windows:
+    - Check [this](https://github.com/git-for-windows/git/wiki/Symbolic-Links) if you do not know how allow non-admins to create symbolic links.
+- Unblock the script
+    - Right-click the script and choose Properties. In the Properties   dialog, if available, select the Unblock checkbox and press OK.
+- If it still doesn’t work now, type the following in a PowerShell window: 
+    - ```Set-ExecutionPolicy -ExecutionPolicy RemoteSigned```
+- Run `symlinks.ps1` from your root space containing `dev-tools` submodule:
+    - ```.\dev-tools.\symlinks.ps1```
```

Comment:
```suggestion
    - ```.\path\to\dev-tools\symlinks.ps1```
```

Or do you need that extra `.`? I don't know PowerShell.

```suggestion
    - ```.\path\to\dev-tools.\symlinks.ps1```
```

Reply:
Yep. We need to use `.`. If no we will get into the error:
![error](https://user-images.githubusercontent.com/68085133/146797802-d2acfa64-7253-41ea-8257-584fc9c5b934.png)

Reply:
If we wanna run script from another folder => than the following command is legal: ```\some\path\script.ps1```.
If we run script from the current folder => we should use `.\`!